documentation/api/query/v4/packages.markdown

```diff
@@ -56,13 +89,13 @@ The array is unsorted by default.
 
 ### Example
 
-[You can use `curl`][curl] to query information about nodes:
+[You can use `curl`][curl] or `puppet query` to query information about nodes:
 
-    curl 'http://localhost:8080/pdb/query/v4' -d  'query=packages{ certname = "agent1" }'
+    puppet query "package_inventory{ certname = 'agent1' }"
 
-    curl http://localhost:8080/pdb/query/v4/packages -d 'query=["=", "certname", "agent1"]'
+    curl -G http://localhost:8080/pdb/query/v4/package-inventory -d 'query=["=", "certname", "agent1"]'
```

Comment:
I think another illustrative example here would be to list all certnames on which a given package name/version are installed (i.e for tracking down vulnerable nodes or whatever)

```
package_inventory[certname]{package_name = "openssl" and version = "1.1.0e" }
```